views/components/segment-links/macro.njk

```diff
@@ -12,3 +12,20 @@
         </div>
     {% endif %}
 {% endmacro %}
+
+
+{% macro tableOfContentLinks(flexibleContent, prefix = null, anchor = 'segment') %}
```

Comment:
Can we make this a new file / directory please? Should only really be one macro inside each of these.

Reply:
Done!